tests/virt/vmfakelib.py

```diff
@@ -53,6 +54,9 @@ def inappropriateDevices(self, ident):
 
     @recorded
     def prune_bitmaps(self, sdUUID, imgUUID, volUUID, baseUUID):
+        if "prune_bitmaps" in self.errors:
+            error = self.errors["prune_bitmaps"]
+            return response.error_raw(error.code, error.msg)
```

Comment:
We can simplify this to:

    error = self.errors.get("prune_bitmaps")
    if error:
        raise ...

This is shorter and does only one dict lookup.